core/services/chainlink/application.go

```diff
@@ -198,6 +201,12 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	restrictedHTTPClient := opts.RestrictedHTTPClient
 	unrestrictedHTTPClient := opts.UnrestrictedHTTPClient
 
+	// Configure and optionally start the audit log forwarder service
+	auditLogger, err := audit.NewAuditLogger(globalLogger, cfg.AuditLogger())
```

Comment:
Overloading the name `AuditLogger` for both `audit.AuditLogger` and `audit.AuditLoggerConfig` seems a little confusing. What about being a little more explicit?
```suggestion
	auditLogger, err := audit.NewAuditLogger(globalLogger, cfg.GetAuditLoggerConfig())
```
Alternatively, you could use an interface for the config type, and then implement it directly via `cfg` so it can just be passed as-is without any extra indirection:
```suggestion
	auditLogger, err := audit.NewAuditLogger(globalLogger, cfg)
```
FWIW This is a little more aligned with the rest of the system, but nbd.

Reply:
Yeah I like this